### DISCUSSION TITLE
Can i change the mac address if yes then how ?

### DISCUSSION THREAD

**Discussion by GeeeCoo** [chris1111/Wireless-USB-Big-Sur-Adapter#341] [Q&A]
### My specs is as follows

- MacOS sequoia 15.1 (opencore)
- intel i5 1035g1  
- wifi adapter is "TPlink wn722n v2"

### The wireless network utility is running flawlessly But i wonder how can i change the mac address for the adapter

**Comment by bielcarreiro** [comment]
Use the Macchanger, this should help you: https://github.com/acrogenesis/macchanger
